app/views/entities/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class = "app_header">
+  <a href= "<%= categories_path %>">
+    <i class="fa-solid fa-arrow-left" id = "left-arrow"></i>
+  </a>
+  <h2> Transactions</h2>
+  <a href = "/logout" id = "logout">Logout</a>
+</div>
+
+<div class="categories">
+  <% if @entities.nil? || @entities.empty?%>
+      <div class="no-record">
+        <p>No Transaction for this Category. </p>
+        <p>Please Add a Transaction</p>
+      </div>
+  <% else %>
+      <div class="total-amount">
+        <p>Total: <%= @category.total_transactions%>$</p>
+      </div>
+      
+      <% @entities.each do |entity| %>
+        <div class = "transaction" >
+          <p><%= entity.name %></p>
+          <p>Price: <%= entity.amount %>$ </p>
+        </div>
+      <% end %>
+  <% end %>
+</div>
+
+<div class ="footer">
+    <i class="fa-solid fa-plus"></i>
+    <%= link_to 'New Transaction', new_category_entity_path, class: "footer_link" %>
+</div>
+
```

Comment:
- [ ] Kindly follow the given design so as to meet the requirements more. Here we need to make a small change.

Given design
![image](https://github.com/Microverse-Fullstack-Program/rails-budget-app-v2/assets/81932478/a4d34dfa-b0d3-498a-848e-1f38cab30bf5)

Your design
![image](https://github.com/Microverse-Fullstack-Program/rails-budget-app-v2/assets/81932478/96b93deb-704c-4cc1-b491-373737ffa146)

- Kindly change the color of the `Add Category` to match the given design
- Kindly ensure the price for each transaction is aligned to the right as is in the given design 

#### Note these changes apply to the categories form page, transactions page and form page as well